web-leaflet/react/README.md

```diff
@@ -0,0 +1,94 @@
+# Using React with Leaflet (and Esri Leaflet)
+
+> if you don't like to read, just check out the [live sample](http://esri.github.io/developer-support/web-leaflet/react/index.html).
+
+## Prerequisites
+
+> "Marc was almost ready to implement his "hello world" React app" - [@thomasfuchs](https://twitter.com/thomasfuchs/status/708675139253174273)
+
+## Development Steps
+
+1. fork/clone this github repository
+2. navigate into this folder using a command prompt and run `npm install`
+> this will lay down the project [dependencies](https://github.com/Esri/developer-support/blob/pages/web-leaflet/react/package.json#L8-L21) in the same folder.
+
+3. run `npm start`
+> afterward you'll be able to check out the sample application running at http://localhost:8080
+
+## Why React?
+
+Why not?  React must be cool.  Its an [open source project](https://facebook.github.io/react/) from Facebook developers.
```

Comment:
i'll make sure we include a sample for it just as soon as its appeal falls past the apex. :stuck_out_tongue_winking_eye: